pipeline/splicing/calcVarPriors.py

```diff
@@ -29,11 +38,69 @@ def getVarStrand(variant):
     if varGene == "BRCA1": 
         return '-'
     elif varGene == "BRCA2":
-        return '+'
+         return '+'
```

Comment:
it's hard to see if the spacing's right here, but make sure it's the correct number!